src/polyhedralGravity/calculation/SanityCheck.cpp

```diff
@@ -0,0 +1,76 @@
+#include "SanityCheck.h"
+
+namespace polyhedralGravity::SanityCheck {
```

Comment:
if this corresponds to the thing that in the readme you refer to as InputCheck maybe we should name it the same for consistency and so we find it later? However, maybe "MeshChecking" or similar would be more precise?